essentials/teleport.dsc

```diff
@@ -0,0 +1,210 @@
+teleport_command:
+  type: command
+  name: teleport
+  debug: false
+  description: Teleports you or another player to a location, to a world, or to another player
+  usage: /teleport help / (player) ((x y z (world)) / (world) / (player))
+  aliases:
+    - tp
+  permission: behrry.essentials.teleport
+  tab complete:
```

Comment:
i recommend zozer's TabCompleteEngine for this mess

Reply:
I do not, his is very procedural and this is just linear based on how many arguments are being used